### ISSUE TITLE
vote rewards not giving correct number of items

### ISSUE THREAD

**Issue by markrjackson** [dmustanger/7dtd-ServerTools#32]
Sorry to submit yet another bug, but my users are reporting that vote rewards sometimes doesn't give the full number of items (I currently have it set to 10 items). Sometimes it gives 10 and other times 2 or 3. Can you look into this?

**Comment by ObsComp** [comment]
Sure. Is this the total entries from the list or the stack counts?

**Comment by markrjackson** [comment]
Total entries not stack counts

**Comment by ObsComp** [comment]
How many entries are there? It does not repeat entries from the list so if you want them to get more of something, up the count.
I will do some testing though in case I am missing something

**Comment by markrjackson** [comment]
I have 17 items in the list and I have config set to give 10 items. My issue is NOT about quantities. The issue is that it doesn't give the correct *NUMBER OF ITEMS* as defined in the config (RewardCount="10"). I just now did a test myself and confirmed this. I did vote reward and got only 1 item (again this is NOT quantities of EACH item but rather the total items themselves). Then I did a "vr" reset in console and did another vote reward and this time I got 3 items (with varying quantities). So there is clearly something wrong because it doesn't give 10 items as I have configured. Just wanted to let you know...
